### PR TITLE
DashR Shiny Movies App

### PR DESCRIPTION
**Pull request by milaroisin**
Issue for app: #177

# App pull request
- [x] I am improving an existing app (redesigns/code "makeovers")

## About

- Playground deployment URL (new version):
- Current gallery app URL: (delete this line if inapplicable)
- Python app repository link: (delete this line if you are working on a Python app)

## Workflow

- [x] I have created a branch in the appropriate monorepo, and the
  elements necessary for successful deployment are in place.
- [x] If the app is a redesigned and/or restyled version of an
  existing gallery app, I've summarized the changes requested in the
  appropriate Streambed issue and confirm that they have been applied.
- [x] If the app is on the Dash Gallery portal, I have added a link to
  the GitHub repository for the source code in the portal description.
- [x] If the app is a reimplementation of a Python gallery app for the
  DashR gallery, the app in this PR mimics, as closely as possible,
  the style and functionality of the existing app.=
- [x] I have removed *all Google Analytics code* from the app's
  `assets/` folder.

## The pre-review review

I have addressed all of the following questions:

- [x] Does everything in my code serve some purpose? (I have removed
  any dead and/or irrelevant code.)
- [x] Does everything in my code have a clear purpose? (My code is
  readable and, where it isn't, it has been commented appropriately.)]
- [x] Am I reinventing the wheel? (I have used appropriate packages to
  lessen the volume of code that needs to be maintained.)

### CODE REVIEW

File: apps/dashr-shiny-movies/app.R [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                 movies,
+                 x = movies$rating,
+                 autobinx = FALSE,
+                 type = "histogram",
+                 xbins = list(
+                   start = minx,
+                   end = maxx,
+                   size = size
+                 )
+               ) %>% layout(xaxis = list(
```
Comment (HammadTheOne) [inline apps/dashr-shiny-movies/app.R]: Getting an error when applying the layout piped in. If I comment out the layout code I am able to run it.

File: apps/dashr-shiny-movies/app.R [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+app$layout(htmlDiv(
+  list(
+    plotlyLogo,
+    pageTitle,
+    firstP,
+    htmlDiv(list(slider, className = "five columns")),
+    htmlBr(),
+    htmlDiv(list
+            (dccGraph
+              (id= "histogram", classname = "histogram"), className = "seven columns"))
```
[the remark this answers is not in the document]
Comment (milaroisin) [reply apps/dashr-shiny-movies/app.R]: Nope, that was

> I'm not sure that the "histogram" class is actually present in your CSS file. Is it supposed to be?

Nope, that was a typo! Thanks!

**Comment by milaroisin** [comment]
@rpkyle PR review and merge please, this is completed

**Comment by CanerIrfanoglu** [comment]
Committed f238432ebae83bf232a9ee38dccdd9d323d35dec. 

The app was already functional before this commit. The changes made mainly improves layout by making it look more alike to the [shiny equivalent](https://plot.ly/r/shinyapp-movies/).

- Horizontally aligned `dccSlider` and `histogram`
- Refactored code for `dccSlider`
- Broke app sections with proper commenting
- Linted the app

**Comment by josegonzalez** [comment]
As per https://github.com/plotly/streambed/issues/13854, I am closing this as the new deployment process is in place.

If you need any changes from this branch, please recreate your changes against master and create a new pull request. Let me know if you need help and I am happy to jump in.

**Pull request closed without merging**